akka-actor/src/main/resources/reference.conf

```diff
@@ -864,12 +884,14 @@ akka {
       selector-dispatcher = "akka.io.pinned-dispatcher"
 
       # Fully qualified config path which holds the dispatcher configuration
-      # for the read/write worker actors
-      worker-dispatcher = "akka.actor.default-dispatcher"
+      # for the read/write worker actors. If left empty (the default), the dispatcher
+      # id from 'akka.actor.internal-dispatcher' is used.
+      worker-dispatcher = ""
```

Comment:
I'd like to keep an explicit setting here instead of documentation. Was it removed for technical reasons?

Reply:
Because it goes via the indirect `akka.actor.internal-dispatcher` setting now instead, to allow a single place for changing what dispatcher is used, my thinking is that there are three scenarios for these:

1. leave it be as is (the new default internal dispatcher)
2. you want to keep using how it worked before, you change `akka.actor.internal-dispatcher` to be the `akka.actor.default-dispatcher` and it will run all dispatcher-configurable internals on that
3. you are the super power user who want to tweak individual dispatchers for things like this, you change this setting (least likely)

Reply:
Another idea would be to let the actual places we create the dispatcher deal with indirection with something along the lines of if the key is a string, follow it to look for an actual dispatcher config, if it is an object it is the dispatcher config.

The most flexible but maybe overcomplicating things?

Reply:
The last suggestion seems difficult to follow. One benefit with the on/off flag is that these settings would by default be
```
worker-dispatcher = akka.actor.default-internal-dispatcher
```

which is straightforward to understand (no indirection nor special `""` value)

Reply:
Ah, ok, so off would then make that the default dispatcher?

Reply:
That was my thinking. We could have that `if` in `Dispatchers.lookup` (or `lookupConfigurator`)

Reply:
After sleeping on it I'm leaning towards that it will be better to just have that full dispatcher name as default in all those individual settings and if you want to run a single dispatcher you'll just have to change all of them.

Reply:
but that would be many (10-20?) and more will be added over time. what is the drawback of providing the flag?

Reply:
Could we improve the config / dispatcher lookup logic in a way that the default could be

```
worker-dispatcher = akka.actor.internal-dispatcher
```

and then to make the internal dispatcher the default just use:

```
akka.actor.internal-dispatcher = akka.actor.default-dispatcher
```

?

Reply:
This feels like the best idea so far to me:

Allow aliases for dispatchers - if `Dispatcher.lookup(id)` finds a string, it follows that string to an actual dispatcher. Change all default dispatchers for internal actors to an explicit `akka.actor.internal-dispatcher`. That setting can then be easily overridden in user config to be an alias pointing to the default (or some other) dispatcher.